.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
     strategy:
       matrix:
-        go-version: [1.17.x]
+        go-version: [1.17.x, 1.18.x, 1.19.x, 1.20.x]
```

Comment:
Linting with each version of Go might be a little pedantic. Lmk your thoughts.

Reply:
No need to support 1.17 and 1.18

Reply:
yeah, let's just keep it to the last 2 supported releases of Go